ddtrace/contrib/asgi/middleware.py

```diff
@@ -70,7 +70,7 @@ def _default_handle_exception_span(exc, span):
 def span_from_scope(scope):
     # type: (Mapping[str, Any]) -> Optional[Span]
     """Retrieve the top-level ASGI span from the scope."""
-    return scope.get("datadog", {}).get("request_span")
+    return scope.get("datadog", {}).get("request_spans")[0]
```

Comment:
we cannot assume that it exists and is a list with items.

Reply:
I changed it, is the way I chose decent or should I do something else?